.coveragerc

```diff
@@ -1051,10 +1050,6 @@ omit =
     homeassistant/components/plex/view.py
     homeassistant/components/plum_lightpad/light.py
     homeassistant/components/pocketcasts/sensor.py
-    homeassistant/components/point/__init__.py
-    homeassistant/components/point/alarm_control_panel.py
-    homeassistant/components/point/binary_sensor.py
-    homeassistant/components/point/sensor.py
```

Comment:
`point` has very low coverage... should it still qualify as `gold`?
```
---------- coverage: platform linux, python 3.12.2-final-0 -----------
Name                                                    Stmts   Miss  Cover   Missing
-------------------------------------------------------------------------------------
homeassistant/components/point/__init__.py                173    112    35%   67-82, 88-119, 124-143, 150-158, 163-172, 182-189, 193, 197-227, 231, 235-237, 241, 246, 250, 254, 264-282, 286, 290-294, 298-299, 307, 312, 317, 322-326, 331, 336
homeassistant/components/point/alarm_control_panel.py      58     58     0%   3-117
homeassistant/components/point/binary_sensor.py            58     58     0%   3-130
homeassistant/components/point/config_flow.py              95     13    86%   89, 94, 112-123, 128, 183-190
homeassistant/components/point/const.py                     8      0   100%
homeassistant/components/point/sensor.py                   28     28     0%   3-87
-------------------------------------------------------------------------------------
TOTAL                                                     420    269    36%
```

Reply:
Let's correct it.

Reply:
Reverted